modules/contentbox-admin/layouts/inc/HTMLBodyEnd.cfm

```diff
@@ -15,7 +15,7 @@
         <script type="application/javascript" src="#prc.cbroot#/includes/js/#js#.js"></script>
     </cfloop>
     <cfloop list="#event.getValue( "jsFullAppendList", "", true )#" index="js">
-        <script type="application/javascript" src="#prc.cbroot#/includes/js/#js#.js"></script>
+        <script type="application/javascript" src="#js#.js"></script>
```

Comment:
That was a separate ticket. Since there might be backwards compat, and I didn't know if you were ok with that or not, I just fixed this bug.
If we want to remove them, we can, but i thought we needed to discuss that first.